massdriver-application-gcp-cloud-run/vpc_connector.tf

```diff
@@ -0,0 +1,15 @@
+# This allows access from Cloud Run to your VPC
+# for database / cache network access
+
+resource "google_vpc_access_connector" "connector" {
+  count          = var.vpc_connector_cidr == null ? 0 : 1
+  name           = module.application.params.md_metadata.name_prefix
+  provider       = google-beta
+  region         = var.location
+  ip_cidr_range  = var.vpc_connector_cidr
```

Comment:
I don't like this part. To access a private database from cloud run, you have to use a VPC connector. As far as I can tell, this can be shared or per-app. Either way it needs a non-conflicting CIDR range. I don't want a user to have to pick a CIDR range to deploy to cloud run but until this lives somewhere else, we have to pass it in.

Reply:
I'm going to try and use the private service connect we set up w/ the VPC in place of this, running a test.

Reply:
The private service connect didn't work. Leaving the connector in here for now. Ideally we can make it so a user doesn't have to pick this CIDR when provisioning a CR app.